.github/workflows/mips-build.yml

```diff
@@ -0,0 +1,55 @@
+name: MIPS Build and Package
```

Comment:
If you add new jobs, add in the [top level README](https://github.com/o1-labs/proof-systems/blob/master/README.md#ci) what this file does, for consistency.

Reply:
Note: not blocking for this PR, but would love to have it at least in a follow-up.